annoyed-alligators/socl_media/templates/terminal.html

```diff
@@ -0,0 +1,112 @@
+  <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
```

Comment:
Just make a sub-directory inside of the templates folder. Once we have a lot of apps, it'll start getting messy to have them all in one place.
So, like `templates/terminal/terminal.html`.
Make sure to change it wherever you're importing it.